gcp/appengine/frontend_handlers.py

```diff
@@ -590,3 +591,8 @@ def list_packages(vuln_affected: list[dict]):
           packages.append(parsed_scheme)
 
   return packages
+
+
+@blueprint.app_errorhandler(404)
+def not_found_error(error: exceptions.HTTPException):
```

Comment:
I think this should placate the 

```
gcp/appengine/frontend_handlers.py:597:20: W0613: Unused argument 'error' (unused-argument)
```

lint finding in https://github.com/google/osv.dev/actions/runs/8730594216/job/23955056146#step:8:79

```suggestion
def not_found_error(unused_error: exceptions.HTTPException):
```

Reply:
I just pushed a change to log the request path. I'm open to go to the suggested path above too.

Reply:
Your approach is superior, stick with that.